### DISCUSSION TITLE
Nested node position offset

### DISCUSSION THREAD

**Discussion by homeless2010** [bcakmakoglu/vue-flow#1721] [Q&A] [answered]
not nested node
![image](https://github.com/user-attachments/assets/3915787a-9fcd-4a6e-a2f4-1b702bc85c98)
position correct (all in one position)
![image](https://github.com/user-attachments/assets/4dbcbb72-456e-4299-9bc2-1280d52e68e9)
nested node
![image](https://github.com/user-attachments/assets/a6eb1727-28a4-4a7f-aed3-90e6011a4809)
![image](https://github.com/user-attachments/assets/1a9d2b21-e4a7-40d6-9a8b-8336501c08c3)
Parent node and child node are offset Large offset

**Comment by bcakmakoglu** [comment, marked as answer]
Your post is lacking a proper description again but I think I can assume what you're asking is why the position seems to behave differently than you expect.
This is due to to the behavior of `position` when a `parentNode` is used.
A node with a parentNode id will have it's position *relative* to the parent node. That means `position: { x: 0, y: 0 }` would be the *top-left* corner of *the parent node*, while a node *without* a parent node will position itself relative to the *viewport*, meaning `position: { x: 0, y: 0 }` would be the top-left corner of the viewport (given it has not been moved whatsoever).

**Reply by homeless2010** [reply]
ok i got it thanks
